README.md

```diff
@@ -1,8 +1,62 @@
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Jasmeet Brar . Jarvis Consulting
 
+I recently graduated from University of Toronto with an Honours Bachelor of Science (Computer Science Specialist and Math Minor), and I intend to venture into software development and data engineering. With my strong background in mathematics, it became natural that I am centered around problem-solving, and being able to construct elegant solutions that are extendable, scalable, and maintainable. In terms of academics, I have studied operating systems, algorithms, computability, web development, parallel programming, scalable computing, machine learning, deep learning, and much more. I have experience in projects involving compilers, web programming, scripting, and data visualization. As for work experience, I had worked as a software developer in helping a firm migrate its frontend services, and I currently work at Jarvis as a data engineer, where I work on projects that involve constructing some pipelines to deliver data in a format that would allow for further analysis. Altogether, they make up all the skills and experience that I want to carry forward towards new opportunities. New opportunities that will make use of my abilities and allow me to further learn new technologies. I am passionate about this journey that I'm on, and I can't wait to see what's up ahead.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Python, C, C++, HTML, JavaScript, TypeScript, React
+
+**Competent:** CSS, PHP, Numpy, NodeJS, Docker, MongoDB
+
+**Familiar:** VueJS, Styled Components, AWS ECS, DynamoDB, GitHub Actions
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar](https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar/tree/master/linux_sql)]: Implemented a monitoring agent that can track the hardware specifications and usage throughout time of host machines and store their data in a database. The monitoring agent is programmed using Bash, and Postgres is used as a database, which would run in a Docker container. Testing was all done on a CentOS 7 virtual machine running on the Google Cloud Platform. Every script was tested manually to ensure that they all adhere to their specifications.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar/tree/master/core_java)]:
+      
+  - Twitter App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - JDBC App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - Grep App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
```

Comment:
don't forget to update those when you finished the project

Reply:
Alright, I updated the descriptions for Grep and JDBC.